content/blog/2018/12/2018-12-10-the-official-Docker-image.adoc

```diff
@@ -0,0 +1,32 @@
+---
+:layout: post
+:title: Official Jenkins image to use from Docker Hub
+:tags:
+- docker
+:author: batmat
+---
+
+There a now three different Docker Hub repositories that are or have been used as the "official" Jenkins image.
```

Comment:
```suggestion
There are now three different Docker Hub repositories that are or have been used as the "official" Jenkins image.
```

Reply:
Followup fix proposed in https://github.com/jenkins-infra/jenkins.io/pull/1957